make docs-serve not working

Cannot run `make docs-serve`, error message:
```
ERROR    -  mkdocstrings: pacti.terms.polyhedra.loaders could not be found
ERROR    -  Error reading page 'Polyhedral.md':
ERROR    -  Could not collect 'pacti.terms.polyhedra.loaders'

Aborted with a BuildError!
✗ Serving documentation (1)
make: *** [docs-serve] Error 1
```
Commenting out the line `::: pacti.terms.polyhedra.loaders` in Polyhedral.md seems to fix it, but I haven't checked if it messes up something else.

--- a/docs/Polyhedral.md
+++ b/docs/Polyhedral.md
@@ -1,2 +1,1 @@
 ::: pacti.terms.polyhedra.polyhedra
-::: pacti.terms.polyhedra.loaders

